src/main/java/net/dv8tion/jda/core/entities/impl/TextChannelImpl.java

```diff
@@ -386,6 +386,23 @@ public JDA getJDA()
     }
 
     @Override
+    public RestAction<Void> clearReactionsByMessageId(String messageId)
+    {
+        checkPermission(Permission.MESSAGE_MANAGE);
+        return new RestAction<Void>(getJDA(), Route.Messages.REMOVE_ALL_REACTIONS.compile(getId(), messageId), null)
```

Comment:
Move the `Route.Messages.REMOVE_ALL_REACTIONS.compile(getId(), messageId)` into its own variable